tests/ludwig/utils/data_utils_test.py

```diff
@@ -0,0 +1,55 @@
+# -*- coding: utf-8 -*-
```

Comment:
based on your suggestion, I renamed all the test modules to test_x.py. Let's stick to that. 

I just checked that the image_utils_test.py is a duplicate file. I will delete it.